README.md

```diff
@@ -1,3 +1,15 @@
 # go-template
 
 golang 工程的模板项目，约定目录结构 以及 基本组件
+
+## 目录结构
+| directory | 描述 |
```

Comment:
style: add a blank line before this line

Reply:
What is the reference basis for this file structure?

Reply:
ref to : https://draveness.me/golang-101/
The style has fixed.